### PR TITLE
Use default runner & Remove the inheritance dependency on the CucumberAndroidJUnitRunner

### PR DESCRIPTION
**Pull request by rhavran**

### CODE REVIEW

**Comment by lsuski** [comment]
I've requested some changes and when they are implemented then It will be merged. I don't have currently time to do this myself and it is not required by any my commercial project. If you can implement required changes in this PR I will merge and release new version.

**Comment by lsuski** [comment]
It is "nice to have" feature but it still requires to run all tests twice with some gradle property to switch between tests type. In my projects I have 2 flavours in dimension `testsType`, e.g. `cucumber` and `androidJunit` with different `testInstrumentationRunner` to have the same effect.

**Comment by DenisBronx** [comment]
@lsuski where can I see the requested changes?

**Comment by lsuski** [comment]
eh, my bad. I've made comments but didn't submit them. Now they should be visible

**Comment by rhavran** [comment]
Closed by accident.

File: cucumber-android/src/main/java/cucumber/runtime/android/CucumberAndroidJUnitArguments.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    private Bundle processedArgs;
+
+    public CucumberAndroidJUnitArguments(@NonNull Bundle arguments) {
+        this.originalArgs = new Bundle(arguments);
+    }
+
+    public Bundle processArgs() {
+        processedArgs = new Bundle(originalArgs);
+
+        if (TRUE.equals(originalArgs.getString(Args.USE_DEFAULT_ANDROID_RUNNER, FALSE))) {
```
Comment (rhavran) [inline cucumber-android/src/main/java/cucumber/runtime/android/CucumberAndroidJUnitArguments.java:67]: @lsuski, passing a boolean property through the "android.testInstrumentationRunnerArguments.cucumberUseAndroidJUnitRunner=true" gives the ClassCastException, so I have to change it to the String, hope it is OK?

**Comment by rhavran** [comment]
Hi @lsuski, it is ready for review now, could you please check?

**Comment by lsuski** [comment]
I'll release it when sonatype.org is up again

**Pull request merged**